apps/community/Makefile

```diff
@@ -1,22 +1,25 @@
-MKFILE_PATH := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+TARGETS = bin/amd64-linux/community \
+		bin/amd64-darwin/community \
+		web
 
-export OUT_DIR := $(MKFILE_PATH)out
+include ../../common.mk
 
-all : clean build-frontend build-backend
+all:: $(TARGETS)
 
-clean :
-	rm -rf $(OUT_DIR)
+clean::
+	rm -rf web
+	rm -rf bin
 
-build-frontend : 
-	rm -rf $(MKFILE_PATH)frontend/build/*
-	mkdir -p $(OUT_DIR)/frontend
-	$(MAKE) -C frontend build-frontend
-	cp -R $(MKFILE_PATH)frontend/build/* $(OUT_DIR)/frontend
+bin/amd64-linux/%:
```

Comment:
hmm i wonder if there's something better to do to build for all OSes besides having to write it out separately for each. esp since its kinda repetitive now that you have `GO_BUILD_AMD64_xxx`

Reply:
Yeah its a bit arbitrary. I think its good to be explicit tho